docs/checkers/index.md

```diff
@@ -2124,6 +2124,30 @@ Example:
 
 ```
 
+By default, there are no Input / output functions ([`input`], [`open`] and [`print`]) allowed.
+However, users may want to specify the permissible functions for utilizing input / output operations.
+To allow a specific function to use input / output functions, use **allowed-io** option.
```

Comment:
Instead of using bold, surround allowed-io with backticks (`) to make it render in code font.

Also, make this sentence more explicit when it comes to types: "use the allowed-io option to specify a list of function names where input/output functions are allowed".

Reply:
Hello Professor Liu,
I was wondering if you can elaborate on what you mean by "more explicit when it comes to types"?
Thanks :)